### PR TITLE
chore: Update disclaimer Spacewalk bridge

### PR DESCRIPTION
**Pull request by Sharqiewicz**
### What:

Update disclaimer Spacewalk bridge

### How:

✅ Update disclaimer text

### CODE REVIEW

**Review by prayagd — CHANGES_REQUESTED** [review]
All good, Just one change in the third bullet i.e in "Total issuable amount (in USD)" please change number of 20000 USD to 50000 USD

**Review by Sharqiewicz — COMMENTED** [review]
Fixed

**Review by annatekl — APPROVED** [review]
good

File: src/pages/bridge/Issue/index.tsx [hunk 1]
```diff
@@ -78,8 +80,15 @@ function Issue(props: IssueProps): JSX.Element {
           >
             here
           </a>
+          .
+        </li>
+        <li>
+          Estimated time for issuing: In a minute after submitting the Stellar payment to the vault, contact
```
Comment (ebma) [inline src/pages/bridge/Issue/index.tsx:86]: How about we end the sentence here. 
```suggestion
          Estimated time for issuing: In a minute after submitting the Stellar payment to the vault. Contact
```

Comment (prayagd) [reply src/pages/bridge/Issue/index.tsx:86]: Agree

**Comment by Sharqiewicz** [comment]
✅ Ready @ebma

**Review by ebma — APPROVED** [review]

**Review by prayagd — APPROVED** [review]
Good to go

**Pull request merged**